scripts/test/Muon/grouping_tab/grouping_table_presenter_test.py

```diff
@@ -358,6 +358,27 @@ def test_disabling_range_max_editing_removes_context_variable(self):
         self.assertFalse('GroupRangeMax' in self.data.gui_variables)
         self.assertEqual(self.gui_variable_observer.update.call_count, 2)
 
+    def test_updating_range_min_to_be_greater_than_range_max_displays_warning_and_vice_versa(self):
+        original_max = '1.12'
+        original_min = '0.1'
+        self.view.group_range_max.setText(original_max)
+        self.view.group_range_min.setText(original_min)
+        self.view.group_range_use_last_data.setChecked(False)
+        self.view.group_range_use_first_good_data.setChecked(False)
+
+        self.view.group_range_min.setText('2.0')
+        self.view.group_range_min.editingFinished.emit()
+
+        self.assertEqual(self.data.gui_variables['GroupRangeMin'], float(original_min))
+        self.view.warning_popup.assert_called_once_with('Minimum of group asymmetry range must be less than maximum')
+
+        self.view.group_range_max.setText('0.05')
+        self.view.group_range_max.editingFinished.emit()
+
+        self.assertEqual(self.data.gui_variables['GroupRangeMax'], float(original_max))
+        self.view.warning_popup.assert_called_with('Maximum of group asymmetry range must be less than minimum')
```

Comment:
This should also say 'Maximum of group asymmetry range must be greater than minimum'